service/controller/resource/tccp/create.go

```diff
@@ -119,26 +117,6 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		}
 
 		if update {
-			err = r.stopMasterInstance(ctx, cr)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-
-			err = r.detachVolumes(ctx, cr)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-
-			err = r.snapshotEtcdVolume(ctx, cr)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-
-			err = r.terminateMasterInstance(ctx, cr)
```

Comment:
I assume we need to revert all these changes for the migration. So I assume you did that for testing reasons.